### PR TITLE
AT_01.02_029 | <Header> Verify case sensitive option in the Search box.

### PR DESCRIPTION
**Pull request by milladiz84**
https://trello.com/c/x5Icfco0/1239-at0102029-header-verify-case-sensitive-option-in-the-search-box

### CODE REVIEW

File: cypress/e2e/headerSearchBox.cy.js [hunk 1]
```diff
@@ -126,4 +127,22 @@ describe('Header Search Box', () => {
         cy.get('a[href="/"].model-link').click();      
     })
   });
+
+  it('AT_01.02_029 | Verify case sensitive option in the Search box', () => {
+    cy.get(`a[href="/user/${userName}"]`).realHover();
+    cy.get('div>a[href^="/user/"]>button[class="jenkins-menu-dropdown-chevron"]').click();
+    cy.get('a[href$="/configure"].yuimenuitemlabel').click()
+    cy.scrollTo(0, 800);
+    cy.get('div.setting-main input[type="checkbox"]').click({ force: true });
```
Comment (Rom-T) [inline cypress/e2e/headerSearchBox.cy.js:136]: Please delete { force: true } and change locator to `label`

File: cypress/e2e/headerSearchBox.cy.js [hunk 2]
```diff
@@ -126,4 +127,22 @@ describe('Header Search Box', () => {
         cy.get('a[href="/"].model-link').click();      
     })
   });
+
+  it('AT_01.02_029 | Verify case sensitive option in the Search box', () => {
+    cy.get(`a[href="/user/${userName}"]`).realHover();
+    cy.get('div>a[href^="/user/"]>button[class="jenkins-menu-dropdown-chevron"]').click();
+    cy.get('a[href$="/configure"].yuimenuitemlabel').click()
+    cy.scrollTo(0, 800);
```
Comment (Rom-T) [inline cypress/e2e/headerSearchBox.cy.js:135]: Don't need. Please delete

File: cypress/e2e/headerSearchBox.cy.js [hunk 3]
```diff
@@ -126,4 +127,22 @@ describe('Header Search Box', () => {
...
+    cy.get('div.setting-main input[type="checkbox"]').click({ force: true });
+    cy.get('button[name="Submit"]').click();    
+    headers.dataCapCase.forEach(arr => {
+      cy.get('input#search-box')
+        .clear()
+        .type(arr + '{enter}')
+        .then(() => {
+          cy.get('input#search-box').clear();
+        })
```
Comment (Rom-T) [inline cypress/e2e/headerSearchBox.cy.js:144]: Should pass without it. Delete please.

Comment (Rom-T) [inline cypress/e2e/headerSearchBox.cy.js:144]: Should pass without it. Delete please.

Comment (milladiz84) [reply cypress/e2e/headerSearchBox.cy.js]: Deleted it, thank you

File: cypress/e2e/headerSearchBox.cy.js (hunk 1 above)
Comment (milladiz84) [reply cypress/e2e/headerSearchBox.cy.js]: Changed it, thank you

**Review by Rom-T — APPROVED** [review]
LGTM

**Review by milia-menzatova — APPROVED** [review]

**Review by MarinaMone — APPROVED** [review]

**Pull request merged**